backend/application/core/models.py

```diff
@@ -544,7 +544,7 @@ class Observation_Log(Model):
     )
     severity = CharField(max_length=12, choices=Severity.SEVERITY_CHOICES, blank=True)
     status = CharField(max_length=16, choices=Status.STATUS_CHOICES, blank=True)
-    comment = CharField(max_length=255)
+    comment = TextField()
```

Comment:
All other fields have a limitation of the length, I would like to see that here as well. Shouldn't 2048 or 4096 characters be enough for a comment?